modules/references/server/controllers/reference.server.controller.js

```diff
@@ -72,8 +73,23 @@ function validateCreate(req) {
     details.userTo = 'userId expected';
   }
 
+  if (_.has(req, ['body', 'feedbackPublic'])) {
+    const { feedbackPublic } = req.body;
```

Comment:
Comment from @simison in https://github.com/Trustroots/trustroots/pull/1384#discussion_r409512920

> What kind of sanitation we should add here?

I borrowed the client side part from the report message, which is plaintext and uses `textService.plainText`, so seems a good option here too.